View/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Space Oddity</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="game">
+        <div class="earth"><img src="../assets/earth.jpg" alt="earth"></div>
+        <div class="atmosphere"></div>
+        <div class= "space"><img src="../assets/space.jpg" alt="space"></div>
+        <div class="landing"><img src="../assets/landing.jpg" alt="earth"></div>
+
+        <div class="spaceship"><img src="../assets/spaceship.png"></div>
+        <div class="spaceship"><img src="../assets/spaceship.png"></div>
+        <div class="spaceship"><img src="../assets/spaceship.png"></div>
+
+        <div class="car"><img src="../assets/tesla.png"></div>
+        <div class="car"><img src="../assets/tesla.png"></div>
+        <div class="car"><img src="../assets/tesla.png"></div>
+        <div class="car"><img src="../assets/tesla.png"></div>
```

Comment:
If you need to write the same line more than three times, you need to use a function to do it.